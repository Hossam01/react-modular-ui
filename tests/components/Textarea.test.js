import React from 'react';
import renderer from 'react-test-renderer';
import Textarea from '../../src/components/Textarea';

describe('Textarea', () => {
  it('should render Textarea', () => {
    const tree = renderer.create(
      <Textarea
        size="128"
        color="primary"
        view="flat"
        disabled
      />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
