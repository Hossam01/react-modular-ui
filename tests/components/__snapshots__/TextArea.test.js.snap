// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea should render TextArea 1`] = `
<div
  className="c-textarea -size-128 -view-flat -color-primary -disabled"
>
  <textarea
    disabled={true}
    name={undefined}
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyPress={[Function]}
    onKeyUp={[Function]}
    placeholder={undefined}
    style={Object {}}
    value={undefined}
  />
</div>
`;

exports[`TextArea should render TextArea with height 1`] = `
<div
  className="c-textarea -size-128 -view-flat -color-primary -disabled"
>
  <textarea
    disabled={true}
    name={undefined}
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyPress={[Function]}
    onKeyUp={[Function]}
    placeholder={undefined}
    style={
      Object {
        "height": "200px",
      }
    }
    value={undefined}
  />
</div>
`;
